### PR TITLE
Custom model training and batch prediction

### PR DESCRIPTION
**Pull request by udaypunna**
New PR: custom_model_training_and_batch_prediction
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "cells": [
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>do not update copyright year</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>move Dataset after Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>add -q option to pip3</p><p> </p><p>remove execution output throughout notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>use notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>remove TODO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>remove future tense : will</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>explain params</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>explain pipeline params</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>explain params</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>remove 'will'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>missing deleting the pipeline job</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

File: notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>This looks like the wrong cell type</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb:2]: <p>This looks like the wrong cell type</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by andrewferlitsch** [comment]
Added more comments. Looks like several code cells got changed to raw cells

**Comment by andrewferlitsch** [comment]
there still appears to be something wrong with [Raw cell] types, though comments show resolved.

**Comment by gericdong** [comment]
@andrewferlitsch I see you have already been reviewing this notebook so I am re-assigning this PR to you. Thanks,

**Comment by udaypunna** [comment]
All the comments addressed into the new pr 873 and it has merged.
PR: https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/873 .

**Pull request closed without merging**